Configuration/Services.yaml

```diff
@@ -0,0 +1,28 @@
+services:
```

Comment:
Please use the PHP version of the config to check if EXT:dashboard is installed before registering the widgets. An example can be found on https://github.com/FriendsOfTYPO3/widgets/blob/master/Configuration/Services.php